docs/troubleshooting.md

```diff
@@ -75,6 +75,25 @@ Run the `heketi-cli server operations info` command and wait until the
 knowing no new operations will be accepted.
 
 
+### Checking database consistency
+
+In older versions of Heketi, database could become inconsistent due to some bugs. To check consistency of the database, the cli variant of the command is `heketi-cli db check` and the server variant of the command is `heketi db check`. These commands only need access to db. They don't collect or compare data with gluster.
+
+
+### Comparing state in heketi database with state of Gluster
+
+Heketi manages Gluster Storage pools and stores the state in its database. In some cases, state of Gluster may diverge from that stored in the database either due to bugs or due to activities performed directly on Gluster Storage pools bypassing Heketi. To summarize such differences, run the `heketi-cli server state examine gluster` command or `heketi offline state examine gluster --config /path/to/heketi.config.json` command. It fetches the following data from Gluster pools:
```

Comment:
"state of Gluster" -> "the state of Gluster"

Reply:
done